src/modules/user/domain/user-update.domain.ts

```diff
@@ -0,0 +1,12 @@
+import { UserEntity } from '../entities/user.entity';
+import { UserUpdateDto } from '../dto/user-update.dto';
+
+export class UserUpdate {
+  public readonly name?: string;
+
+  static toEntity(userUpdateDto: UserUpdateDto): Partial<UserEntity> {
```

Comment:
```suggestion
  static toEntity(userUpdate: UserUpdate): Partial<UserEntity> {
```

Từ domain sang entity